### DISCUSSION TITLE
Weather station ID sources

### DISCUSSION THREAD

**Discussion by KeeperOfMaps** [CosmoQuestX/Rocket-Bot#13] [General]
From someone I know at ECCC:
_

> From the website: [Home | World Weather Information Service](https://worldweather.wmo.int/en/home.html)
> 
> I clicked on Support and selected Download which brought me to:[Download | World Weather Information Service](https://worldweather.wmo.int/en/dataguide.html)
> 
> And selecting option 1. I go to: https://worldweather.wmo.int/en/json/full_city_list.txt

_

Also this:

> Oh, I also found this page that might help: [World Weather Records](https://www.ncei.noaa.gov/access/metadata/landing-page/bin/iso?id=gov.noaa.ncdc:C00160)

**Comment by KeeperOfMaps** [comment]
Look what arrived in my email this morning...
[swob-xml_station_list.csv](https://github.com/CosmoQuestX/Rocket-Bot/files/9337675/swob-xml_station_list.csv)

**Comment by TheRealFakeAdmin** [comment]
Could this issue be changed to a discussion?

Adding to this, could we enable the discussions feature for this repo?

**Comment by KeeperOfMaps** [comment]
I'll try to remember to do that next time I'm in front of a computer.
